fearless/Modules/NewWallet/WalletTransactionHistory/WalletTransactionHistoryViewFactory.swift

```diff
@@ -93,4 +84,37 @@ enum WalletTransactionHistoryViewFactory {
             items: filters
         )]
     }
+
+    private static func createHistoryDeps(
```

Comment:
actually historyoperationfactory used across the app, not only inside this module. because of that, it's better to have some Assembly that can generate concrete implementation of history operation factory based on chain. I'm already implemented this assembly so it will be replaced once we merged our branches together. nothing to do here right now.